src/components/FeaturePanel/ImageSection/Photo.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { Image } from '../../../services/types';
+
+interface PhotoInnerProps {
+  image: Image;
+}
+
+const InnerPano: React.FC<PhotoInnerProps> = ({ image }) => {
+  const [url, setUrl] = React.useState('');
+  const [pannellumUrl, setPannellumUrl] = React.useState<string | null>(null);
+
+  React.useEffect(() => {
+    setUrl(`${window.location.protocol}//${window.location.host}`);
+  }, []);
+
+  React.useEffect(() => {
+    setPannellumUrl(
+      `https://cdn.pannellum.org/2.5/pannellum.htm#panorama=${encodeURIComponent(
+        image.thumb,
+      )}&config=${encodeURIComponent(`${url}/pannellum-config.json`)}`,
+    );
+  }, [url]);
```

Comment:
Can we perhaps merge it into one useEffect? The `url` state wouldn't be needed.

```suggestion
  const [pannellumUrl, setPannellumUrl] = React.useState<string | null>(null);

  React.useEffect(() => {
    const url = `${window.location.protocol}//${window.location.host}`
    setPannellumUrl(
      `https://cdn.pannellum.org/2.5/pannellum.htm#panorama=${encodeURIComponent(
        image.thumb,
      )}&config=${encodeURIComponent(`${url}/pannellum-config.json`)}`,
    );
  }, [url]);
```